### PR TITLE
Do not execute lint on empty source list

### PR DESCRIPTION
**Pull request by rovner**
Hi, we have multi module project and some modules are lack of scss sources by design. Current behivor is to execute lint for those modules what cause maven build to fail.

### CODE REVIEW

File: src/main/java/nl/geodienstencentrum/maven/plugin/sass/Resource.java [hunk 1]
```diff
@@ -62,7 +62,7 @@
 		final Map<String, String> result = new LinkedHashMap<>();
 
 		if (!sourceDirectory.exists()) {
-			log.error("Specified sourcedirectory (" + sourceDirectory
+			log.warn("Specified sourcedirectory (" + sourceDirectory
```
Comment (mprins) [inline src/main/java/nl/geodienstencentrum/maven/plugin/sass/Resource.java]: actually this is an error in the configuration

**Comment by mprins** [comment]
It seems you have configured the plugin at a super pom level, so the behaviour is actually correct; however this seems like a sensible addition

**Comment by mprins** [comment]
merged in #222

**Comment by rovner** [comment]
@mprins Thanx!
Just for clarification, we have 200+ maven modules, so I have configured plugin in parent pom intentionally. Configuring each module looks like way to hell :)
One more question, when I can expect this to be released?

**Pull request closed without merging**